org.eclipse.corrosion/src/org/eclipse/corrosion/debug/RustDebugDelegate.java

```diff
@@ -159,6 +159,12 @@
 		wc.setAttribute(ICDTLaunchConfigurationConstants.ATTR_PROGRAM_NAME, executable.getAbsolutePath());
 		wc.setAttribute(ICDTLaunchConfigurationConstants.ATTR_WORKING_DIRECTORY, workingDirectory.getAbsolutePath());
 		wc.setAttribute(ICDTLaunchConfigurationConstants.ATTR_LOCATION, project.getLocation().toString());
+		
+		String stopInMainSymbol = configuration.getAttribute(ICDTLaunchConfigurationConstants.ATTR_DEBUGGER_STOP_AT_MAIN_SYMBOL, ""); //$NON-NLS-1$
+		if (false == projectName.equals(".cargo") && (stopInMainSymbol.equals("main"))) { //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
You can simply ignore the case of the .cargo project at the moment, it's not meant to be used for debug anyway

Reply:
Done